proposals/csharp-10.0/lambda-improvements.md

```diff
@@ -187,19 +222,19 @@ The [first phase](../../spec/expressions.md#the-first-phase) allows an anonymous
 > *  Otherwise, type inference fails.
 
 ### Best common type
-Best common type is defined in terms of type inference (see [finding the best common type](../../spec/expressions.md#finding-the-best-common-type-of-a-set-of-expressions)) so the changes above apply to best common type as well.
+[Best common type](../../spec/expressions.md#finding-the-best-common-type-of-a-set-of-expressions) is defined in terms of type inference so the type inference changes above apply to best common type as well.
 ```csharp
 var fs = new[] { (string s) => s.Length; (string s) => int.Parse(s) } // Func<string, int>[]
 ```
 
 ### `var`
-Lambda expressions and method groups with natural types can be used as initializers in `var` declarations.
+Anonymous functions and method groups with inferred types can be used as initializers in `var` declarations.
```

Comment:
>inferred [](http://example.com/codeflow?start=43&length=8)

"... with function types" would be better.
We have a section on `## Natural (function) type`, we didn't talk about "inferred function type" or "inferred type" before in the doc.